wpm/commandline.py

```diff
@@ -275,7 +278,7 @@ def main():
     try:
         with wpm.game.GameManager(quotes, stats, opts.cpm) as gm:
             try:
-                gm.run(to_front=text_ids)
+                gm.run(to_front=text_ids, hard_flag=opts.hard)
```

Comment:
As stated below, I think it's better to just store this in the GameManager instance than to pass it around.